conformance/src/test/scala/za/co/absa/enceladus/conformance/standardization/stages/TypeParserSuiteTemplate.scala

```diff
@@ -13,20 +13,21 @@
  * limitations under the License.
  */
 
-package za.co.absa.enceladus.standardization.interpreter.stages
+package za.co.absa.enceladus.conformance.standardization.stages
 
 import java.security.InvalidParameterException
 import java.sql.{Date, Timestamp}
 
 import org.apache.log4j.{LogManager, Logger}
 import org.apache.spark.sql.types._
 import org.scalatest.FunSuite
-import za.co.absa.enceladus.standardization.interpreter.dataTypes.ParseOutput
-import za.co.absa.enceladus.utils.types.{Defaults, GlobalDefaults, TypedStructField}
-import za.co.absa.enceladus.standardization.interpreter.stages.TypeParserSuiteTemplate._
+import za.co.absa.enceladus.conformance.standardization.dataTypes.ParseOutput
+import za.co.absa.enceladus.conformance.standardization.stages.TypeParser
```

Comment:
`import za.co.absa.enceladus.conformance.standardization.stages.TypeParser` is unused and produces a _Warning_